llvm/lib/Transforms/Vectorize/VPlanRecipes.cpp

```diff
@@ -880,6 +888,8 @@ bool VPInstruction::onlyFirstLaneUsed(const VPValue *Op) const {
     return false;
   case Instruction::PHI:
     return true;
+  case Instruction::ExtractElement:
+    return Op == getOperand(1);
```

Comment:
```suggestion
    return false;
 case Instruction::ExtractElement:
    return Op == getOperand(1);
  case Instruction::PHI:
    return true;
```

nit: might be good to loosely try to keep things ordered